### ISSUE TITLE
MiterClip reverts to Bevel

### ISSUE THREAD

**Issue by turnage** [nical/lyon#573]
MiterClip behaves the same as Miter when the miter limit is reached.

[Here is an animation](https://user-images.githubusercontent.com/2957168/80286818-7ad91780-86e2-11ea-8378-11e6e09ab1a6.gif) of a curve with LineJoin::MiterClip increasing in thickness, each frame tessellated as a stroke.

The specific path is:

```
Move to: (150.0, 200.0)
Cubic: control 0 (150.0, 210.0) control 1 (191.2, 257.5) end (248.29828, 506.05005)
Cubic: control 0 (311.2, 257.5) control 1 (350.0, 210.0) end (350.0, 200.0)
```

**Comment by nical** [comment]
Thanks for reporting this. The miter-clip code was just very wrong. The fix is up in version 0.15.8.

**Comment by turnage** [comment]
Thank you for all your hard work, Nical! It’s valuable to many people.

Vào CN, 26 thg 4, 2020 lúc 7:28 SA Nicolas Silva <notifications@github.com>
đã viết:
